### PR TITLE
UI: Develop Homepage - Get In Touch section added

### LINKED ISSUE #214: UI: Develop Homepage - Get In Touch section
## Description
As a maintainer,
I need to see BIT pages redesigned,
so that they meet the AnitaB Org guidelines.

## Mocks
Approved design idea for `Get in touch form` section (Designer Rahul Mohata)
Screen Shot 2021-03-28 at 5 23 59 pm

### Update [Required]
- [ ] Implement the design above in the section it is intended (Get In Touch section of the homepage).

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1 hour

## Additional Information

Link to [Figma file](https://www.figma.com/file/V22SXBLzMd9bqnita6hfMp/Bridge_in_tech?node-id=113%3A20)

### PR DESCRIPTION
**Pull request by shades-7**
### Description
Added Get in Touch section

Fixes #214 

### Type of Change:
- Code
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
BIT (1)

### Checklist:

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by codesankalp** [comment]
@shades-7 Can you make the PR title more descriptive?

**Comment by decon-harsh** [comment]
@shades-7 You wrote the GetinTouch() function but you haven't linked it in home page right ? This is my home-page.

Screenshot from 2021-04-14 15-12-45

Let me know if I am wrong , new to frontend development :laughing:

**Comment by shades-7** [comment]
> @shades-7 You wrote the GetinTouch() function but you haven't linked it in home page right ? This is my home-page.
> 
> Screenshot from 2021-04-14 15-12-45
> 
> Let me know if I am wrong , new to frontend development 😆

Yes I was not sure should I make changes to routes.js or not I just made the component and I thought it will require just one import to check it. 
I can add routes.js file if u want

**Comment by shades-7** [comment]
> @shades-7 Can you make the PR title more descriptive?

hey I am not sure is it possible to change the PR title after opening it, can you help me with this?

**Comment by decon-harsh** [comment]
> > @shades-7 You wrote the GetinTouch() function but you haven't linked it in home page right ? This is my home-page.
> > 
> > Screenshot from 2021-04-14 15-12-45
> > 
> > Let me know if I am wrong , new to frontend development 😆
> 
> Yes I was not sure should I make changes to routes.js or not I just made the component and I thought it will require just one import to check it. 
> I can add routes.js file if u want

Yes please , cause at the end you have to link it up in the home page right ?

**Comment by shades-7** [comment]
@decon-harsh done

**Comment by decon-harsh** [comment]
> > @shades-7 Can you make the PR title more descriptive?
> 
> hey I am not sure is it possible to change the PR title after opening it, can you help me with this?

Yes you can , you might have an edit option right to the title, like this.

Screenshot from 2021-04-14 15-57-30

**Review by Rahulm2310 — COMMENTED** [review]
@shades-7 `Get In Touch` section should be added to `home.jsx` i.e. home page. Please remove it from the `Routes.js` file.

File: src/Getintouch/getintouch.jsx
Comment (Rahulm2310) [inline src/Getintouch/getintouch.jsx]: @shades-7 Please use a generic name like `John Doe` and a generic email `johndoe@gmail.com`.

File: src/Getintouch/getintouch.module.css
Comment (Rahulm2310) [inline src/Getintouch/getintouch.module.css]: Are these the preffered media query break points?

Comment (shades-7) [reply src/Getintouch/getintouch.module.css]: I was just decreasing the font size so that it will look good even on the smallest phone screen.

File: src/Getintouch/getintouch.jsx
Comment (shades-7) [reply src/Getintouch/getintouch.jsx]: Okay will update soon

File: src/home/Home.jsx
Comment (Rahulm2310) [inline src/home/Home.jsx:7]: Please add proper formatting to the jsx.

Comment (shades-7) [reply src/home/Home.jsx:7]: I'm sorry I didn't get this, can you please elaborate?

Comment (Rahulm2310) [reply src/home/Home.jsx:7]: I mean add proper indentation

File: src/home/Home.jsx
Comment (codesankalp) [inline src/home/Home.jsx]:
```suggestion
<section id="home">
    <div className="header-text">
        <h1>Welcome to<br /> Bridge-in-Tech</h1>
        <p>Bridge-In-Tech is an application inspired by the existing
            AnitaB.org Mentorship System. It encourages organizations to collaborate
            with the mentors and mentees on mentoring programs. Through Bridge-In-Tech,
            an organization can offer a mentorship program to a mentor and a mentee that is customised
            to meet the needed skills set within its organisation while providing a safety and supportive
            environment for these mentor/mentee to work in.
        </p>
    </div>
</section>
```

This is what @Rahulm2310 wants to say @shades-7

Comment (Rahulm2310) [reply src/home/Home.jsx]: yup! Thanks @codesankalp

Comment (shades-7) [reply src/home/Home.jsx]: hey @mtreacy002  can you review my PR?

Comment (mtreacy002) [reply src/home/Home.jsx]: will look into it 😉

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@shades-7 , the section looks great. I just need to point out few things:

- we need to maintain the files naming convention. So, please change the following:
    - folder name `Getintouch` to use camel case.
    - the .css/.jsx files to start with capital letters. `i` in `in` and `t` in `touch` also need to be capitals.
    - I also don't think we need to use `.module` for the .css, just `GetInTouch.css` will do. 

image

- Please place the form on the left and the section info on the right, as per the approved design on the figma file.
    Here's how the section is approved on the figma file:
Screen Shot 2021-03-28 at 5 23 59 pm
    Here's how the PR section looks like:
    
image

**Comment by shades-7** [comment]
@mtreacy002 made changes as per request

**Review by mtreacy002 — CHANGES_REQUESTED** [review]
@shades-7 , can you please remove the unused folder `Getintouch` and its files?

Screen Shot 2021-04-24 at 10 12 20 am

This also seems to be related to this frontend error

Screen Shot 2021-04-24 at 10 20 00 am

**Comment by shades-7** [comment]
> @shades-7 , can you please remove the unused folder `Getintouch` and its files?
> 
> Screen Shot 2021-04-24 at 10 12 20 am
> 
> This also seems to be related to this frontend error
> 
> Screen Shot 2021-04-24 at 10 20 00 am

Fixed

**Comment by mtreacy002** [comment]
@shades-7 ? I still see the deleted folder in the commit and your fork repo pr branch? May I know what you did to try remove them?

Screen Shot 2021-04-24 at 11 56 34 am
Screen Shot 2021-04-24 at 11 56 53 am

**Comment by shades-7** [comment]
> @shades-7 ? I still see the deleted folder in the commit and your fork repo pr branch? May I know what you did to try remove them?
> 
> Screen Shot 2021-04-24 at 11 56 34 am
> 
> Screen Shot 2021-04-24 at 11 56 53 am

I am so sorry for the inconvenience. I believe I have done this time. @mtreacy002

**Review by mtreacy002 — APPROVED** [review]
LGTM, now @shades-7 .
I've tested this PR locally and here's the test report:
1. Code review: Done
2. Tested on all possible outcomes.
    Expected behaviour: The new `Get In Touch` section should be present and responsive to various screen size.
    Actual behaviour: As expected (see gif below).
ezgif com-gif-maker (49)

3. Environments
    OS: MacOS 10.15.7
    Browser: Google Chrome

Thank you again for your contribution to the project, @shades-7 🎉🎉🎉
PS: I just want to request approval from @Rahulm2310 as OSH mentors before I can merge this to develop.

**Review by Rahulm2310 — APPROVED** [review]
Good work @shades-7 💯🎉. @mtreacy002 I think it is ready to merge now ☺️

**Review by decon-harsh — APPROVED** [review]

**Pull request merged**
